test/ApiClient.test.ts

```diff
@@ -284,6 +286,9 @@ describe('Subscribe', () => {
     const unsubscribeFn = client.subscribe(req, cb)
     await sleep(1200)
     expect(numEnvelopes).toBe(2)
+    // Resubscribing triggers an info log
+    expect(consoleInfo).toBeCalledTimes(1)
```

Comment:
Also somehow lost this comment. I don't think we should be relying on expecting console logs for a test. In the future, someone could very reasonably remove the log statement and add another one elsewhere, such that behavior changes but the tests still pass.

Reply:
I see, it's not a test requirement but just meant to suppress the console logs. In that case looks good to me.